tests/unit/amo/components/TestSponsoredAddonsShelf.js

```diff
@@ -64,6 +65,7 @@ describe(__filename, () => {
   };
 
   const _loadPromotedExtensions = ({ addons = [] }) => {
+    store.dispatch(setLang('en-US'));
```

Comment:
`dispatchClientMetadata()` (which is called in the test setup method) already dispatches this action. Why is this needed here?

Reply:
You're right. I added that prematurely. I'll remove it.